test/integration/backward_compatible/parallel/serialization/compact/CompactTest.js

```diff
@@ -78,7 +78,7 @@ describe('CompactTest', function () {
     before(async function () {
         TestUtil.markClientVersionAtLeast(this, '5.1.0');
         const comparisonValueForServerVersion520 = await TestUtil.compareServerVersionWithRC(RC, '5.2.0');
-        if ((await TestUtil.compareServerVersionWithRC(RC, '5.1.0')) < 0) {
+        if ((await TestUtil.compareServerVersionWithRC(RC, '5.2.0')) < 0) {
```

Comment:
I think this should stay like the previous version, and we should add a check below. Something like 
```
        // Compact serialization 5.2 and newer clients are not compatible with servers older than 5.2
        if (comparisonValueForServerVersion520 < 0 && TestUtil.isClientVersionAtLeast('5.2.0')) {
            this.skip();
        }
```

Reply:
Fixed